WorkingInR/2_First_Steps_in_R.Rmd

```diff
@@ -271,7 +285,8 @@ Read more about vectors [here](https://www.rdocumentation.org/packages/base/vers
 
 ### Factors
 
-A factor is a vector which elements may be organized in levels (i.e. category labels), which comes handy when running a statistical analysis that includes categorical variables. Data elements stored in a factor are usually numeric and/or character, and vectors containing numeric and/or character data may easily be converted to factors using `as.factor()`.
+A factor is a vector which elements may be organized in levels (i.e. category labels), which comes handy when running a statistical analysis that includes categorical variables.
+Data elements stored in a factor are usually numeric and/or character, and vectors containing numeric and/or character data may easily be converted to factors using `as.factor()`.
```

Comment:
Recommend using `factor` rather than `as.factor` as it has arguments for levels and labels etc